dir2cast.php

```diff
@@ -1027,12 +1112,13 @@ protected function scan()
                 $item_count = $this->addItem($file->getPath() . '/' . $file->getFileName());
             }
             
-            if(0 == $item_count)
-                throw new Exception("No Items found in {$this->source_dir}");
+            // if(0 == $item_count)
```

Comment:
Do you want to bring this back?

Reply:
Actually no, I was considering removing it completely, or perhaps making it configurable. It has the effect of making it impossible to test an "empty" podcast has the right structure, as it triggers the error page. The error page is probably more user-friendly, so thanks for prompting me to think about this!